lib/pure/asyncnet.nim

```diff
@@ -780,9 +780,9 @@ proc isClosed*(socket: AsyncSocket): bool =
   ## Determines whether the socket has been closed.
   return socket.closed
 
-proc sendTo*(socket: AsyncSocket, data: pointer, dataSize: int, address: string,
-             port: Port, flags = {SocketFlag.SafeDisconn}): owned(Future[void])
-            {.async, since: (1, 3).} =
+proc sendTo*(socket: AsyncSocket, data: string, address: string, port: Port,
```

Comment:
Please change order of arguments so that it matches the ones in `net`: https://nim-lang.org/docs/net.html#sendTo%2CSocket%2Cstring%2CPort%2Cstring

Reply:
It's done.